### PR TITLE
Learn Sensu (Quick Start Guide)

### PR DESCRIPTION
**Pull request by calebhailey**
Cherry picked from #379.

### CODE REVIEW

**Comment by cwjohnston** [comment]
I like the look of this so far! 👍

**Comment by calebhailey** [comment]
We should revive this effort and publish this alongside the upcoming 0.27 release. /cc @cwjohnston

**Comment by cwjohnston** [comment]
Eventually this needs to be a number of separate pages instead of a one-shot scroll-fest

**Comment by cwjohnston** [comment]
And  to match this guide we need to update the Five Minute Sensu to use Centos 7

File: docs/0.28/quick-start/the-five-minute-install-centos.md [hunk 1]
```diff
@@ -0,0 +1,243 @@
...
+   Sensu Enterprise as well:
+
+   ~~~ shell
+   sudo yum install sensu-enterprise sensu-enterprise-dashboard
+   ~~~
+
+5. Configure Sensu by downloading this [example configuration file][10]:
+
+   ~~~ shell
+   sudo curl -o /etc/sensu/config.json https://sensuapp.org/docs/0.28/files/simple-sensu-config.json
```
Comment (cwjohnston) [inline docs/0.28/quick-start/the-five-minute-install-centos.md]: Consider replacing curl usage here with `echo '{ "transport": { "name": "redis" } }' > /etc/sensu/config.json`

File: docs/0.28/quick-start/the-five-minute-install-centos.md [hunk 2]
```diff
@@ -0,0 +1,243 @@
...
+       "port": 4567
+     }
+   }
+   ~~~
+
+6. Configure the Sensu client by downloading this [example configuration
+   file][11]:
+
+   ~~~ shell
+   sudo curl -o /etc/sensu/conf.d/client.json https://sensuapp.org/docs/0.28/files/simple-client-config.json
```
Comment (cwjohnston) [inline docs/0.28/quick-start/the-five-minute-install-centos.md]: As of Sensu 0.26 client will auto-detect ip address and name.

Consider replacing curl usage here with `echo '{ "client": { "environment": "development", "subscriptions": ["dev"] } }' > /etc/sensu/conf.d/client.json`

File: docs/0.28/quick-start/the-five-minute-install-centos.md [hunk 3]
```diff
@@ -0,0 +1,243 @@
...
+       }
+     }
+   }
+   ~~~
+
+7. Configure a Sensu dashboard by downloading this [example configuration
+   file][12]:
+
+   ~~~ shell
+   sudo curl -o /etc/sensu/dashboard.json https://sensuapp.org/docs/0.28/files/simple-dashboard-config.json
```
Comment (cwjohnston) [inline docs/0.28/quick-start/the-five-minute-install-centos.md]: Consider removing this curl usage in favor of the simple file content below.

File: docs/0.28/quick-start/the-five-minute-install-centos.md [hunk 4]
```diff
@@ -0,0 +1,243 @@
...
+   sudo systemctl start sensu-enterprise.service
+   sudo systemctl start sensu-enterprise-dashboard.service
+   sudo systemctl start sensu-client.service
+   ~~~
+
+10. Verify that your installation is ready to use by querying the Sensu API
+    using the `curl` utility (and piping the result to the [`jq` utility][13]):
+
+    ~~~ shell
+    sudo yum install jq -y
```
Comment (cwjohnston) [inline docs/0.28/quick-start/the-five-minute-install-centos.md]: Consider adding `curl` to the install command here so we're sure to have both.

**Comment by calebhailey** [comment]
Wow! This is awesome progress! Nice work guys. :punch:

**Pull request merged**